base_classes/NXdetector.nxdl.xml

```diff
@@ -705,13 +705,16 @@
     </dimensions>
   </field>
   <field name="gain_setting" type="NX_CHAR">
-    <doc>The gain setting of the detector. This influences background etc.</doc>
-    <enumeration>
-      <item value="high"/>
-      <item value="standard"/>
-      <item value="fast"/>
-      <item value="auto"/>
-    </enumeration>
+    <doc>
+      The gain setting of the detector. This is a detector-specific value
+      meant to document the gain setting of the detector during data
+      collection, for detectors with multiple available gain settings.
+
+      Examples of gain settings include: standard, fast, auto, high,
+      medium, low, mixed high/medium, and mixed medium/low. Developers are
```

Comment:
Changes from NIAC review
- Change / to -
- Change list to bullets, with quotes around terms
- Developers are recommended to use one of these terms (list).  Copy language from elsewhere that encourages users to submit additional terms.

Reply:
Similarly in MXmx: https://github.com/nexusformat/definitions/blob/d28e0aa58959fde96b1d748b748229528cf9e670/applications/NXmx.nxdl.xml#L661-L664

Reply:
Also convert indentation to spaces, not tabs